CMakeLists.txt

```diff
@@ -155,10 +156,10 @@ if(SIMENG_ENABLE_TESTS)
 
       # Print message containing if the full test suite will run
     if (${LLVM_PACKAGE_VERSION} VERSION_LESS "14.0")
-      message(STATUS "LLVM version does not support AArch64 extensions SME or SVE2. These test suites will be skipped.")
+      message(STATUS "LLVM version does not support AArch64 extensions SVE2, SVE2.1, SME, or SME2. Related tests will fail.")
```

Comment:
Not entirely sure on the behaviour of message().  Would LLVM versions less than 14 result in both messages being printed?  If so, could we use an if/else logic to avoid duplicated information?

Reply:
Yup, you're correct - have updated to ifelse